numpy/random/src/philox/philox.h

```diff
@@ -33,7 +33,7 @@ static NPY_INLINE uint64_t mulhilo64(uint64_t a, uint64_t b, uint64_t *hip) {
   return (uint64_t)product;
 }
 #else
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(_M_ARM64)
 #include <intrin.h>
 #if defined(_WIN64) && defined(_M_AMD64)
 #pragma intrinsic(_umul128)
```

Comment:
Is `_umul128` available of ARM for Windows? Is `__emulu` available?

Reply:
Looks like neither is available for ARM.  I think you need to make sure that you hit the path starting at like 69.

Reply:
Yes, those intrinsics weren't available. But I think we could probably use `__umulh` which is available for ARM64 targets as well.

https://docs.microsoft.com/en-us/cpp/intrinsics/umulh?view=msvc-160

Reply:
You could add an implementation with that for ARM.  Right now it looks like you end up in the block that starts at line 40 which needs `__emulu`, rather than the default block at 69 that uses only standard operations.

Reply:
This has been added to latest commit